Raycasting/assets/basicShader.sh

```diff
@@ -0,0 +1,19 @@
+#shader vertex
```

Comment:
this is a good shader to keep around because of its simplicity

Reply:
what is the point of making a single comment if other people can just reply to it anyways?!

Reply:
This comment is a review, see whats different?

Reply:
ohhhh! a review gets added to the conversation and has to be resolved!